mirgecom/simutil.py

```diff
@@ -490,6 +500,9 @@ def compare_files_vtu(
 
     nfields = point_data1.GetNumberOfArrays()
     max_field_errors = [0 for _ in range(nfields)]
+    field_tol = {}  # tolerance dict here for now
```

Comment:
Is there a plan for exposing this to the user so he can specify it?

Reply:
I was under the impression that the user can have the choice to specify individual fields instead of providing just one default value for all fields.

Reply:
We were thinking maybe a field specific tolerance file.

Reply:
I think that sounds like a reasonable option.  Any ideas about format? 

A command-line interface might be handy, too.   It looks like it might be possible (although perhaps ill-advised; @matthiasdiener ?), to do something like:

```python
python mirgecompare.py -t '{"u":  1e-5, "v": 1e-6}' <red_file> <blue_file>
```

Just looking at:

https://stackoverflow.com/questions/18006161/how-to-pass-dictionary-as-command-line-argument-to-python-script

Reply:
I think that format for the command line looks optimal. I'll try to implement that.